examples/webgl_multiple_elements_with_text.html

```diff
@@ -0,0 +1,429 @@
+				var size = .25;
+
+				var colors = [ 'rgb(0,127,255)', 'rgb(255,0,0)', 'rgb(0,255,0)', 'rgb(0,255,255)',
+        				       'rgb(255,0,255)', 'rgb(255,0,127)', 'rgb(255,255,0)', 'rgb(0,255,127)' ]; 
+
+				canvas = document.getElementById( 'c' );
+
+				renderer = new THREE.WebGLRenderer( { canvas: canvas, antialias: true } );
+				renderer.setClearColor( 0xffffff, 1 );
+				renderer.setPixelRatio( window.devicePixelRatio );
+
+				views = document.querySelectorAll( '.view' );
+
+				for ( var n = 0 ; n < views.length ; n++ ) {
+
+					var scene = new THREE.Scene();
+
+					var geometry = new THREE.Geometry();
+					var geometry0 = new THREE.Geometry();
+
+					if ( views[n].lattice ) {
+
+						var range = balls / 2;
+						for ( var i = -range ; i <= range ; i++ ) {
+
+							for ( var j = -range ; j <= range ; j++ ) {
+
+								for ( var k = -range ; k <= range ; k++ ) {
+
+									geometry.vertices.push( new THREE.Vector3( i, j, k ) );
+									geometry0.vertices.push( new THREE.Vector3( i, j, k ) );
+
+								}
+
+							}
+
+						}
+ 
+					} else {
+
+						for ( var m = 0 ; m < Math.pow( balls, 3 ) ; m++ ) {
+
+							var i = balls * Math.random() - balls / 2;
+							var j = balls * Math.random() - balls / 2;
+							var k = balls * Math.random() - balls / 2;
+
+							geometry.vertices.push( new THREE.Vector3( i, j, k ) );
+							geometry0.vertices.push( new THREE.Vector3( i, j, k ) );
+
+						}
+
+					}
+
+					var index = Math.floor( colors.length * Math.random() );
+
+					var canvas2 = document.createElement( 'canvas' );
+					canvas2.width = 128;
+					canvas2.height = 128;
+					var context = canvas2.getContext( '2d' );
+					context.arc( 64, 64, 64, 0, 2 * Math.PI );
+					context.fillStyle = colors[ index ];
+					context.fill();
+					var texture = new THREE.Texture( canvas2 );
+					texture.needsUpdate = true;
+
+					var material = new THREE.PointsMaterial( { size: size, map: texture, transparent: true, alphaTest: .1 } );
+
+					scene.add( new THREE.Points( geometry, material ) );
+
+					scene.userData.view = views[n];
+					scene.userData.geometry0 = geometry0;
+
+					var camera = new THREE.PerspectiveCamera( 75, 1, .1, 100 );
+					camera.position.set( 0, 0, 1.2*balls );
+					scene.userData.camera = camera;
+
+					var controls = new THREE.OrbitControls( camera, views[n] );
+					scene.userData.controls = controls;
+	
+					scenes.push( scene );
+
+				}
+
+				t = 0;
+				animate();
+
+			}
+
+			function updateSize() {
+
+				var width = canvas.clientWidth;
+				var height = canvas.clientHeight;
+
+				if ( canvas.width !== width || canvas.height != height ) {
+
+					renderer.setSize( width, height, false );
+
+				}
+
+			}
+			
+			function animate() {
+			
+				render();
+				requestAnimationFrame( animate );
+				
+			}
+
+			function render() {
+
+				updateSize();
+
+				renderer.setClearColor( 0xffffff );
+				renderer.setViewport( 0, 0, canvas.clientWidth, canvas.clientHeight );
```

Comment:
I still do not think `setViewport()` is needed, as discussed [here](https://github.com/mrdoob/three.js/commit/07a609988f84aad95afa810e5bdef1ff21ea3fea).